src/DssSpell.t.sol

```diff
@@ -615,17 +615,47 @@ contract DssSpellTest is DssSpellTestBase {
 
     RwaLiquidationLike internal oracle = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
 
-    address rwaOperator_008                = 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439;
-    GemAbstract rwagem_008                 = GemAbstract(addr.addr("RWA008"));
-    GemJoinAbstract rwajoin_008            = GemJoinAbstract(addr.addr("MCD_JOIN_RWA008_A"));
-    RwaUrnLike rwaurn_008                  = RwaUrnLike(addr.addr("RWA008_A_URN"));
-    RwaInputConduitLike rwaconduitin_008   = RwaInputConduitLike(addr.addr("RWA008_A_INPUT_CONDUIT"));
-    RwaOutputConduitLike rwaconduitout_008 = RwaOutputConduitLike(addr.addr("RWA008_A_OUTPUT_CONDUIT"));
-
-    GemAbstract rwagem_009         = GemAbstract(addr.addr("RWA009"));
-    GemJoinAbstract rwajoin_009    = GemJoinAbstract(addr.addr("MCD_JOIN_RWA009_A"));
-    RwaUrnLike rwaurn_009          = RwaUrnLike(addr.addr("RWA009_A_URN"));
-    address RWA009_GENESIS_ADDRESS = addr.addr("RWA009_A_OUTPUT_CONDUIT");
+    address internal RWA008_A_OPERATOR              = 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439;
```

Comment:
In a test it doesn't really matter if it is `internal` or not as we don't care the size of the bytecode due the extra getters. It feels the `internal` actually makes the code a bit harder to read. Anyway just a comment, feel free to leave as it is if you prefer.

Reply:
Nevermind, I added this by mistake.